### ISSUE TITLE
ToolbarOptions with SelectableText is not working as expected

### ISSUE THREAD

**Issue by erdoganbavas** [flutter/flutter#42593]
## Steps to Reproduce
1. Start a new flutter application
2. Swap [this](https://gist.github.com/erdoganbavas/3dc191b22ffa2972c64a849b5deb5505) widget with main.dart
3. Run application
4. Select text, "Select All" parameter has false but appears.

**Target Platform:**
Android
**Target OS version/browser:**
Android 5 (Samsung Note 3 Physical device)
Android 9 (Nexus one emulator)
**Devices:**

## Logs

```
[√] Flutter (Channel stable, v1.9.1+hotfix.4, on Microsoft Windows [Version 10.0.18362.388], locale tr-TR)
    • Flutter version 1.9.1+hotfix.4 at C:\src\flutter
    • Framework revision cc949a8e8b (2 weeks ago), 2019-09-27 15:04:59 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0


[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at C:\Users\Erdogan\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[√] Android Studio (version 3.4)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 37.0.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[√] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 9 (API 28) (emulator)

• No issues found!
```

**Comment by HansMuller** [comment]
CC @justinmc

**Comment by justinmc** [comment]
I don't see "Select All" when I try this on a physical Pixel 2.  Am I doing this right?

![flutter_01](https://user-images.githubusercontent.com/389558/66764318-5dd3aa00-ee5e-11e9-9706-856bf2de1a7f.png)

CC @chunhtai

**Comment by erdoganbavas** [comment]
My flutter installation is on "stable" channel. This issue already solved on "master" branch #40713 .

If you want to check your channel execute "flutter channel".

**Comment by justinmc** [comment]
Oh got it!  Thanks for following up.
